sites/rcvfortexas-1/rankit_polls.html

```diff
@@ -0,0 +1,23 @@
+{% include "breadcrumbs" %}
+{% if page.headline.size > 0 and page.is_homepage? == false %}
+<h2 class="headline">{{ page.headline }}</h2>
+{% endif %}
+<div id="content">
+   <div id="intro" class="intro">
+      <div class="text-content">
+         {{ page.basic.content }}
+         <ol class="try-rcv-list">
+            <li>Rank kinds of pie:<br /><br />
+               <div style="position:relative;overflow:hidden;padding-top:120%;">      
+                  <iframe src="https://rankit.vote/vote/BQT1rJWfqJpTN82NLsLE" style="border:1px solid #f1f1f1;position: absolute;top: 0;left: 0;width: 100%;height: 100%;" name="myiFrame" scrolling="yes" frameborder="1" marginheight="0px" marginwidth="0px" allowfullscreen></iframe>
```

Comment:
So, I tested this and, after voting, the page just reloads instead of navigating to the results page (like it does when not in an iframe). Before I knew you were working on this issue, I made a local test page with nothing but an iframe and observed the same behavior.  Since you took over, I didn't really look into the cause, but I'm a little concerned that the rankit.vote folks might have to change something on their end for this to work properly. I don't think it makes sense for this to go live if people can't see the results.

Reply:
Ah okay, I read your email again. I'll withdraw my issue for now.